sysvinit/freebsd/cloudinitlocal

```diff
@@ -18,6 +18,11 @@ start_cmd="cloudlocal_start"
 cloudlocal_start()
 {
 	echo -n "${command} starting"
+	if kenv -q kernel_options | grep 'cloud-init=disabled'; then
+		echo "cloud-init is disabled via /proc/cmdline."
```

Comment:
instead of echo, this should be "warn"

and i should change the other echos in these files to be "info"

Reply:
Also i would say kernel_options instead of /proc/cmdline here

Reply:
> instead of echo, this should be "warn"

does this apply on the netbsd side too?

Reply:
warn, yes.
kenv and kernel_options: nope